package.json

```diff
@@ -27,9 +27,9 @@
   "readme": "README.md",
   "homepage": "https://github.com/auth0/auth0-deploy-cli#readme",
   "dependencies": {
-    "auth0": "^2.23.0",
+    "auth0": "^2.27.0",
     "auth0-extension-tools": "^1.4.4",
-    "auth0-source-control-extension-tools": "^4.0.7",
```

Comment:
TODO: bump to `4.1.1` when https://github.com/auth0-extensions/auth0-source-control-extension-tools/pull/99 is merged.

Reply:
TODO: switch to tilde-matching (`~4.1.1`). Otherwise, new features supported in auth0-source-control-extensions-tools (minor bump) could get automatically pulled by users of auth0-deploy-cli. That can lead to surprising behaviors.

Reply:
Good !